src/systems/filecoin_blockchain/struct/block/_index.md

```diff
@@ -7,5 +7,66 @@ The Block is a unit of the Filecoin blockchain.
 
 {{< readfile file="block.id" code="true" lang="go" >}}
 
-{{< readfile file="block.go" code="true" lang="go" >}}
+{{< readfile file="election.id" code="true" lang="go" >}}
+
+# Block syntax validation
+
+Syntax validation refers to validation that may be performed on a block and its messages 
+without reference to outside information such as the parent state tree.
+
+An invalid block must not be transmitted or referenced as a parent.
+
+A syntactically valid block header must decode into fields matching the type definition below. 
+
+A syntactically valid header must have:
+
+- a non-empty array of `Parents` CIDs if `Epoch` is greater than zero,
+- a non-negative `ParentWeight`,
+- a `Miner` address which is an ID-address,
+- a non-negative `Epoch`,
+- a positive `Timestamp`,
+- a `Ticket` with non-empty `VRFResult`,
+- an `ElectionProof` with non-empty `VRFResult`.
```

Comment:
could you please change the `ElectionProof` bit given new ePoSt work merged in #705 
```suggestion
- `ElectionPoStOutput` containing:
  - a non-empty `Candidates` array
  - a non-empty `PoStRandomness` field
  - a non-empty `Proof` field
```